src/toolbar/toolbar-group.jsx

```diff
@@ -197,7 +196,7 @@ const ToolbarGroup = React.createClass({
     }, this);
 
     return (
-      <div {...other} className={className} style={this.prepareStyles(styles.root, style)}>
+      <div {...other} className={className} style={prepareStyles(Object.assign({}, styles.root, style))}>
```

Comment:
I you're doing this almost everywhere you use `prepareStyles` maybe it's time to move the logic before we merge these? @oliviertassinari What do you think?